saged/datasets.py

```diff
@@ -342,9 +343,9 @@ def get_all_data(self) -> Tuple[np.array, np.array]:
 
         Returns
         -------
-        X: The gene expression data in a genes x samples array
+        X: The gene expression data in a samples x genes array
```

Comment:
That's definitely a good point. I currently have the internal representation stored as genes x samples because that's how they're stored in the compendium file. Since my most common internal operations are subsetting by sample that should be more efficient. Both sklearn and pytorch expect their inputs to be samples x features though, so I take the transpose when returning results to keep the data consistent with what the models are expecting